Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.AdvancedDataGridView/Controls Toolkit/MenuStrip.cs

```diff
@@ -1645,9 +1603,9 @@ private bool HasNodesChecked(TreeNodeItemSelector[] nodes)
                 return state;
             }
 
-            foreach (TreeNodeItemSelector node in nodes)
+            foreach (TreeNodeItemSelector? node in nodes)
```

Comment:
`node` could be null
Therfore line 1608 is going to blow

Reply:
Undo `?`